package.json

```diff
@@ -35,19 +38,39 @@
   "homepage": "https://github.com/MrJacz/discord.js-lavalink#readme",
   "devDependencies": {
     "@types/node": "^12.7.1",
+    "@types/rollup-plugin-progress": "^1.1.0",
     "@types/ws": "^6.0.2",
     "@typescript-eslint/eslint-plugin": "^1.13.0",
     "@typescript-eslint/parser": "^1.13.0",
     "discord.js": "discordjs/discord.js",
     "eslint": "^6.1.0",
     "eslint-plugin-node": "^9.1.0",
     "eslint-plugin-promise": "^4.2.1",
+    "rollup": "^1.19.4",
+    "rollup-plugin-cleaner": "^1.0.0",
+    "rollup-plugin-commonjs": "^10.0.2",
+    "rollup-plugin-copy": "^3.1.0",
+    "rollup-plugin-node-builtins": "^2.1.2",
+    "rollup-plugin-node-globals": "^1.4.0",
+    "rollup-plugin-node-resolve": "^5.2.0",
+    "rollup-plugin-peer-deps-external": "^2.2.0",
+    "rollup-plugin-progress": "^1.1.1",
+    "rollup-plugin-terser": "5.1.1",
+    "rollup-plugin-typescript2": "^0.22.1",
+    "ts-node": "^8.3.0",
     "typescript": "^3.5.3"
   },
   "dependencies": {
-    "ws": "^7.1.1"
+    "ws": "^7.1.2"
   },
   "peerDependencies": {
-    "discord.js": "^11.5.1"
+    "discord.js": "^11"
+  },
+  "files": [
```

Comment:
I would like to include the LICENCE and readme

Reply:
License, package, readme and changelog are always included. Even if you add them to npmignore. And they're still included now (just see the second dry run)

Reply:
Sidenote about that dry run, I see it holds a TS buildinfo file. It didn't do that in later dry runs. Not sure why it landed in dist before.